opentelekomcloud/common/validators.go

```diff
@@ -260,35 +260,6 @@ func ValidateAntiDdosAppTypeID(v interface{}, k string) (ws []string, errors []e
 	return
 }
 
-func ValidateTags(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
Remove the validation because there are very many rules for tag, and different rules for different situations, so it is best to just leave it to the server to validate.